src/esp32/src/uart.c

```diff
@@ -4,10 +4,12 @@
  * SPDX-License-Identifier: Apache-2.0 OR MIT
  */
 
+#include <target/uart.h>
```

Comment:
Please follow the include order as suggested here.
https://docs.espressif.com/projects/esp-idf/en/stable/esp32/contribute/style-guide.html#include-statements

Reply:
Good point. I keep thinking that in C++ the corresponding header comes first. I'll fix